s2i/s2i.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: s2i
+spec:
+  inputs:
+    resources:
+     - name: git-source
+       type: git
+    params:
+    - name: BUILDER_IMAGE
+      description: The location of the builder image.
+      default: docker.io/kbaig/s2i
+    - name: S2I_IMAGE
+      description: The location of the s2i builder image.
+      default: docker.io/centos/nodejs-8-centos7
+
+  outputs:
+    resources:
+    - name: image
+      type: image
+
+  steps:
+  - name: generate
+    image: ${inputs.params.BUILDER_IMAGE}
+    workingdir: /workspace/git-source
+    command: ['s2i', 'build', '.', '${inputs.params.S2I_IMAGE}', '--as-dockerfile', '/gen-source/Dockerfile.gen']
```

Comment:
should we accept the thitd parameter as an `input.parameter`? where the default parameter value would be `.` (current dir)?

So the user could use this task without making a change in task definition?